app/uk/gov/hmrc/bindingtariffclassification/search/Search.scala

```diff
@@ -62,6 +64,11 @@ object Sort {
     SortDirection.values.find(_.toString == key)
   }
 
+  private def defaultSortDirection: SortField => SortDirection = {
```

Comment:
depending on the field we want to sort with (days elapsed or commodity code at the moment), we have a different default sort direction

Reply:
The backend shouldn’t care about this though as it is a Operator UI concern. let the UI specify sort direction.